cluster/images/OWNERS

```diff
@@ -1,6 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
+  - cpanato
```

Comment:
can you please add `# Release Manager` so we know that this is for their role as a release manager?

unless we can count on folks outside of their release manager role... then we would be looking at how many files/directories they have reviewed in the past etc.

same is true for the other files in this PR as well. actually it may be better to just use the alias from `OWNERS_ALIASES` as well i think.

cc @spiffxp @BenTheElder @cblecker

Reply:
> can you please add `# Release Manager` so we know that this is for their role as a release manager?

Fixed!

> same is true for the other files in this PR as well. actually it may be better to just use the alias from `OWNERS_ALIASES` as well i think.

IIRC, the last time I made changes in some of these directories, reviewers preferred explicitly naming the contributors instead of aliases. I prefer aliases, but I'll wait for a few folks to check in before making additional changes.

Reply:
@justaugustus IMHO, if folks are interested in becoming reviewers, then they should be adding themselves as such. here in this PR, it's a bulk update for a specific reason namely "release managers should be aware of what is happening" more than "i am interested in becoming a reviewer" ..

+1 to wait for others to chime in.

Reply:
> if folks are interested in becoming reviewers, then they should be adding themselves as such

Completely agreed! :)